### ISSUE TITLE
Auto updater is not working on macOS

### ISSUE THREAD

**Issue by akinofftz** [meetfranz/franz#1183]
**Describe the bug**
Automatically updating Franz 5 is not working anymore since a few beta version. Update notification is displayed at the bottom, but clicking the button just closes Franz without updating. Opening Franz manually just starts the program, with update notification appearing again.

**To Reproduce**
1. Click on update button in update notification on button.
2. Nothing happens except closing Franz.
3. Open Franz manually.
4. Old version is started and update notification appears again.

**Expected behavior**
Franz should download and install update, notification should disappear after restart.

**Desktop (please complete the following information):**
 - macOS Mojave 10.14.2 (18C54)

**Comment by adlk** [comment]
Do you have Franz installed in the `/Applications` folder?

**Comment by akinofftz** [comment]
Yes. First installation was made by Homebrew Cask, but I successfully updated a few times since then.

**Comment by adlk** [comment]
Hmm... this is definitely something we need to watch. Thanks for reporting!

**Comment by naquiroz** [comment]
Same bug here, on linux.
